ray-operator/config/crd/bases/ray.io_rayclusters.yaml

```diff
@@ -10987,6 +11001,20 @@ spec:
                   claimed by the user at the cluster level.
                 format: int32
                 type: integer
+              headStatus:
+                properties:
+                  detectedRayResources:
+                    additionalProperties:
+                      format: int64
+                      type: integer
+                    description: DetectedRayResources is the map of resource name
+                      to Ray resource capacity for the members of the hea
+                    type: object
```

Comment:
Based on my earlier comments, I think its best not to have this

Reply:
Yeah, I agree -- it is possible to arrange the sequence of events so that the autoscaler always has this info, but it's not worth the brain strain.

The PR implementing the autoscaler computation is here:
https://github.com/ray-project/ray/pull/26219